dbos_transact/context.py

```diff
@@ -8,4 +16,245 @@ def __init__(self) -> None:
         self.executor_id = os.environ.get("DBOS__VMID", "local")
         self.app_version = os.environ.get("DBOS__APPVERSION", "")
         self.app_id = os.environ.get("DBOS__APPID", "")
+
         self.logger = dbos_logger
+
+        self.function_id: int = -1
+
+        self.next_workflow_uuid: str = ""
+        self.workflow_uuid: str = ""
+
+        self.curr_comm_function_id: int = -1
+        self.curr_tx_function_id: int = -1
+        self.sql_session: Optional[Session] = None
+
+    def create_child(self) -> DBOSContext:
+        rv = DBOSContext()
+        rv.logger = self.logger
+        rv.next_workflow_uuid = self.next_workflow_uuid
```

Comment:
we probably don't want to pass in the UUID to child.. When creating a child workflow, the workflow should be able to assign a new UUID.